README.md

```diff
@@ -260,6 +266,7 @@ We can optimise how people collaborate with each other if we align teams with co
 As a starting point, we recommend visualising sociotechnical architecture with the [Context Maps](https://speakerdeck.com/mploed/visualizing-sociotechnical-architectures-with-context-maps). A brief overview of the most important patterns can be found under the [context-mapping](https://github.com/ddd-crew/context-mapping) GitHub Project.
 
 ![Context Map](resources/context-map-cheat-sheet.png)*Credit: Michael Plöd*
+More details and templates about context maps can be found in [this repository](https://github.com/ddd-crew/context-mapping)
```

Comment:
This link appears to be a duplicate of the context mapping link above the image.